src/locales/en-US/stream.ftl

```diff
@@ -196,6 +218,78 @@ profile-settings-download-comments-recentRequest =
 profile-settings-download-comments-timeOut =
   You can submit another request in { framework-timeago-time }
 
+profile-settings-deleteAccount-title = Delete My Account
+profile-settings-deleteAccount-description =
+  Deleting your account will permanently erase your profile and remove
+  all your comments from this site.
+profile-settings-deleteAccount-requestDelete = Request account deletion
+
+profile-settings-deleteAccount-cancelDelete-description =
+  You have already submitted a request to delete your account.
+  Your account will be deleted on { $date }.
+  You may cancel the request until that time.
+profile-settings-deleteAccount-cancelDelete = Cancel account deletion request
+
+profile-settings-deleteAccount-pages-deleteButton = Delete my account
+profile-settings-deleteAccount-pages-cancel = Cancel
+profile-settings-deleteAccount-pages-proceed = Proceed
+profile-settings-deleteAccount-pages-done = Done
+
+profile-settings-deleteAccount-pages-descriptionHeader = Delete my account?
+profile-settings-deleteAccount-pages-descriptionText =
+  You are attempting to delete your account. This means:
+profile-settings-deleteAccount-pages-list1 =
+  All of your comments are removed from this site
+profile-settings-deleteAccount-pages-list2 =
+  All of your comments are deleted from our database
+profile-settings-deleteAccount-pages-list3 =
+  Your username and email address are removed from our system
```

Comment:
Confirm with sam on copy.

Reply:
Sent Sam some screenshots and explained we don't delete usernames.
Updated the message to say we delete email address pre-emptively so that we can get this in and tweak later when Sam can get back to us.